tests/parsing/CliParsingTest.py

```diff
@@ -52,3 +55,12 @@ def test_parse_cli(self):
         self.assertEqual(parsed_sections["default"].name, "Default")
         self.assertEqual(self.dict_from_sections(parsed_sections),
                          expected_dict)
+
+    def test_check_conflicts(self):
+        sections = parse_cli(arg_list=["--save", "--no-config"])
+        with self.assertRaises(SystemExit) as cm:
+            check_conflicts(sections, self.log_printer)
+        self.assertEqual(cm.exception.code, 2)
```

Comment:
I was skeptical myself until I saw an official pytest docs example. I'll make the change.